guides/_subsections/view-targets-with-topology.md

```diff
@@ -0,0 +1,112 @@
+
+{% include_relative _subsections/common/navigate-to-topology.md %}
+
+### [View all target JVMs](#view-all-target-jvms)
+
+<ol>
+    {% capture topology-graphview-guide-text %}
+    By default, an interactive <b>Graph View</b> of discovered target JVMs (<i>nodes</i>) are shown nested their associated groups (<i>surrounding lines</i>), for example, Pod, or Realm (i.e. discovery platform). 
+    <br><br>
+
+    You can drag target nodes/groups or use the bottom control bar to adjust the graph, for example, zoom out or fit all nodes into view. A toolbar is also available to allow further customization:
+
+    <ol>
+        <li>
+            <b>Display Options:</b> adjust how the nodes and groups are displayed, for example, whether to show connection URL.
+        </li>
+        <li>
+            <b>Filters:</b> determine which targets or groups to show.
+        </li>
+        <li>
+            <b>Search bar:</b> find a target using <b>Match Expression</b>. The matched targets will be highlighted.
+        </li>
+    </ol>
+    {% endcapture %}
+    {% capture topology-listview-guide-text %}
+    Topology View also supports <b>List View</b> mode, where your JVM targets and their groups are shown as expandable rows. 
+    <br><br>
+    Click the List icon on the toolbar to switch to <b>List View</b>. Expand each row to see nested groups or targets. All above features of the toolbar can also be used to customize your view.
+    {% endcapture %}
+
+    {% include howto_step.html
+      summary="View JVM applications with Graph View"
+      image-name="2.3.0/topology-1.png"
+      caption="View JVM applications with topology Graph view."
+      text=topology-graphview-guide-text
+    %}
+    {% include howto_step.html
+      summary="View JVM applications with List View"
+      image-name="2.3.0/topology-2.png"
+      caption="View JVM applications with Topology List view."
+      text=topology-listview-guide-text
+    %}
+</ol>
+
+### [View JVM details and resources](#view-jvm-details-and-resources)
+<ol>
+
+    {% capture sidebar-detail-guide-text %}
+    In <b>Graph View</b>, select a target JVM node to open the drawer panel that shows its details, for example, connection URL, labels and annotations. Click <i>Show more</i> to the JVM’s Mbean details.
+    <br><br>
+    In <b>List View</b>, expand each row to open nested groups until you find the target. Expand the target to see its details and associated resources.
+
+    {% endcapture %}
+
+    {% capture sidebar-resource-guide-text %}
+        Navigate to the <b>Resources</b> tab to see the target’s associated resources. There are 2 tables:
+    <ol>
+        <li>
+        <b>Owned Resources:</b> Resources that the Target owns (i.e. active/archived recordings, event templates and event types).</li>
```

Comment:
Target is capitalized here, but not in previous articles, I think we should keep it consistent.